src/Utils/Hexagons.svelte

```diff
@@ -21,39 +28,55 @@ let moveable;
 $: mouseDown = false;
 $: rotation = 30;
 
-const frame = {rotate: 0,};
+$: Xstart = 0;
+$: Ystart = 0;
+$: Xend = 0;
+$: Yend = 0;
+$: deltaX = Xend - Xstart;
+$: deltaY = Yend - Ystart;
+
+const frame = {rotate: 0, translate: [0,0], scale: [1,1]};
+
+onMount(() => {boundRect = target.getBoundingClientRect();});
 
-function getBytesForActuator(actuator) {
+function getBytesForActuator(actuators) {
     /*Passed active actuator, Returns Array of 16 strings with 8 'bits'*/
     let shiftActuator;
     let binActuator;
+    let allActuator = 0;
     const zero = '0';
-    if (actuator <= 31) {
-        shiftActuator = Math.abs(1 << actuator); 
-        binActuator = shiftActuator.toString(2).padStart(32,'0'); 
-        binActuator = zero.repeat(96) + binActuator; //Pad with leading/trailing zeros to fill out to 128 'bits'
-    } else if (actuator >= 32 && actuator < 63) {
-        shiftActuator = Math.abs(1 << (actuator - 32)); 
-        binActuator = shiftActuator.toString(2).padStart(32,'0');
-        binActuator = zero.repeat(64) + binActuator + zero.repeat(32);
-    } else if (actuator >= 64 && actuator < 95) {
-        shiftActuator = Math.abs(1 << (actuator - 64)); 
-        binActuator = shiftActuator.toString(2).padStart(32,'0');
-        binActuator = zero.repeat(32) + binActuator + zero.repeat(64);
-    } else if (actuator >= 96 && actuator < 127) {
-        shiftActuator = Math.abs(1 << (actuator - 96)); 
-        binActuator = shiftActuator.toString(2).padStart(32,'0');
-        binActuator = binActuator + zero.repeat(96);
+    for (let actuator of actuators) {
+        if (actuator <= 31) {
+            shiftActuator = Math.abs(1 << actuator); 
+            allActuator |= shiftActuator;
+            binActuator = Math.abs(allActuator).toString(2).padStart(32,'0'); 
+            binActuator = zero.repeat(96) + binActuator; //Pad with leading/trailing zeros to fill out to 128 'bits'
+        } else if (actuator >= 32 && actuator < 63) {
+            shiftActuator = Math.abs(1 << (actuator - 32)); 
+            allActuator |= shiftActuator;
+            binActuator = Math.abs(allActuator).toString(2).padStart(32,'0'); 
+            binActuator = zero.repeat(64) + binActuator + zero.repeat(32);
+        } else if (actuator >= 64 && actuator < 95) {
+            shiftActuator = Math.abs(1 << (actuator - 64)); 
+            allActuator |= shiftActuator;
+            binActuator = Math.abs(allActuator).toString(2).padStart(32,'0'); 
+            binActuator = zero.repeat(32) + binActuator + zero.repeat(64);
+        } else if (actuator >= 96 && actuator < 127) {
+            shiftActuator = Math.abs(1 << (actuator - 96)); 
+            allActuator |= shiftActuator;
+            binActuator = Math.abs(allActuator).toString(2).padStart(32,'0'); 
+            binActuator = binActuator + zero.repeat(96);
+        }
```

Comment:
If we have any performance problems, this needs to not use strings.

This approach will only work when multi-touch is within the same actuator block. 
* Instead, a correct approach with this style of accumulation would keep a blockNActuator where we accumulate `blockNActuator |= shiftActuator`. 
* Rather than using the current approach, we might as well
```
// Given actuators = [26, 126]
// Produce [
//     0,0,0,0b00000010,
//     0,0,0,0,
//     0,0,0,0b01000000
// ]
let blockBytes = [...Array(16)].keys().map(i => 0);
for(let actuator of actuators) {
  const block  = Math.floor(actuator / 32)
  const blockByte = Math.floor((actuator % 32) / 8)
  blockBytes[(block * 32) + blockByte] |= ((actuator % 32) - (blockByte * 8))
}
return blockBytes
```